nana/modules/animelist.py

```diff
@@ -281,6 +287,49 @@ async def manga_search(client, message):
                 await client.send_photo(message.chat.id, photo=image, caption=ms_g)
             except:
                 ms_g += f" [〽️]({image})"
-                await msg(message, text=ms_g)
+                await edrep(message, text=ms_g)
         else:
-            await msg(message, text=ms_g)
+            await edrep(message, text=ms_g)
+
+
+@app.on_message(filters.user(AdminSettings) & filters.command("favourite", Command))
+async def favourite_animelist(client, message):
+    x = await client.get_inline_bot_results(f"{BotUsername}", f"favourite")
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)